src/download.jl

```diff
@@ -20,21 +18,20 @@ end
 
 Registers and downloads the desired dataset.
 """
-function download(dataset_code)
-    dataset_url = dataurl(dataset_code)
-
+function download(dataid)
+    # TODO: Implement a `isvalid()` function to validate dataid
     # This uses register function of DataDeps package
     # Registers the dataset to be used
     register(
         DataDep(
-            dataset_code,
-            "GADM dataset for $dataset_code",
-            dataset_url,
+            dataid,
+            "GADM dataset for $dataid",
+            dataurl(dataid),
             post_fetch_method=DataDeps.unpack
         )
     )
 
     # downloading the dataset if not available
-    @info "Successfully downloaded the files: ", readdir(@datadep_str dataset_code; join=true)
+    @info "Successfully downloaded the files: ", readdir(@datadep_str dataid; join=true)
```

Comment:
Quick question, doesn't DataDep already print some info message like this at the end?

Reply:
No Datadeps doesn't print anything once download finishes.